opm/simulators/wells/MultisegmentWellSegments.cpp

```diff
@@ -637,8 +637,9 @@ pressureDropSpiralICD(const int seg,
     using MathTool = MathToolbox<EvalWell>;
 
     const double density_cali = sicd.densityCalibration();
-    const EvalWell temp_value1 = MathTool::pow(density / density_cali, 0.75);
-    const EvalWell temp_value2 = MathTool::pow(mixture_viscosity / viscosity_cali, 0.25);
+    // make sure we don't pass negative base to powers
+    const EvalWell temp_value1 = density > 0.0 ? MathTool::pow(density / density_cali, 0.75) : 0.0;
+    const EvalWell temp_value2 = mixture_viscosity > 0.0 ? MathTool::pow(mixture_viscosity / viscosity_cali, 0.25) : 0.0;
```

Comment:
I take it we're guaranteed that the `*_cali` values are always strictly positive?  If not, is there a check somewhere else?  If not, should we add a check here?